src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/ExposureSubmissionTestResultViewModel.swift

```diff
@@ -163,6 +163,7 @@ class ExposureSubmissionTestResultViewModel {
 				self?.error = error
 			case let .success(testResult):
 				self?.testResult = testResult
+				self?.updateWarnOthers()
```

Comment:
if this is the only caller why use a separte func?

Reply:
The reason is to keep it logically separated